renovate.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: renovate
+  version: 37.6.3
+  epoch: 0
+  description: "Automated dependency updates. Multi-platform and multi-language."
+  copyright:
+    - license: AGPL-3.0
+  checks:
+    disabled:
+      - usrlocal
+  dependencies:
+    runtime:
+      - nodejs
+      - git
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+
+pipeline:
+  - name: npm install
+    runs: |
+      npm install -g ${{package.name}}@${{package.version}} -prefix ${{targets.destdir}}/usr/local/
+
+      # https://github.com/browserify/resolve/issues/288
```

Comment:
Same as 
https://github.com/wolfi-dev/os/pull/6409/files#r1349344680
Curious what people think?

Reply:
Thanks James, was chatting with Dan on the other thread and I agree creating the advisory is a good move too, will send one separately!
Sounds like we'd also (maybe temporarily?) like to do the `sed` too though from the other thread, please let me know if you think that's a bad idea and I'll adjust?